document/document.go

```diff
@@ -30,9 +30,18 @@ func init() {
 }
 
 type Document struct {
-	id              string  `json:"id"`
-	Fields          []Field `json:"fields"`
-	CompositeFields []*CompositeField
+	id                string  `json:"id"`
+	Fields            []Field `json:"fields"`
+	CompositeFields   []*CompositeField
+	storedFieldsBytes uint64
+}
+
+func (d *Document) SetStoredFieldsBytes(val uint64) {
```

Comment:
Drop this method you can updated storedFieldsBytes by simply exporting it (like CompositeFields above).

Reply:
Done.